readme.md

```diff
@@ -12,69 +12,59 @@ This is the official ZURB Template for use with [Foundation for Sites](http://fo
 - Built-in BrowserSync server
 - For production builds:
   - CSS compression
-  - JavaScript compression
+  - JavaScript module bundling with webpack
   - Image compression
 
 ## Installation
 
 To use this template, your computer needs:
 
-- [NodeJS](https://nodejs.org/en/) (Up to Version 8 only, Tested with 8.12.0)
+- [NodeJS](https://nodejs.org/en/) (Version 6 or greater recommended, tested with 6.11.4 and 8.12.0)
 - [Git](https://git-scm.com/)
 
-### Setup
```

Comment:
this section should also be in here. This shows the manual setup way.

Reply:
Download and installation instructions are still there, moved to `### Manual Setup` below.